docs/monitoring/inventory.md

```diff
@@ -1,13 +1,105 @@
 title: Sematext Inventory Monitoring
 description: Sematext Inventory Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
 
-This should explain the new Inventory Monitoring feature, and have the following sections.
+The [Sematext Agent](../agents/sematext-agent) provides a simple and versatile way of gathering machine-related information such as host, VM, or container properties, kernel versions, and installed packages. It presents them on a per-host basis allowing you not only to view the data but also search and compare different hosts. All of this, in the same place, shipped automatically without any effort from an operations point of view.
 
-- overview
-- how it works
-- how to enable
-- what it does
-- what it solves 
-- how it helps with observability/troubleshooting/debugging/etc…
+The Inventory information is available in the Monitoring *Infrastructure* section of the *Monitoring* tab of your Sematext Cloud account - your main place for hosts, virtual machines and containers information. 
 
-Inventory monitoring collects info about packages installed on your infrastructure.  It also detects their (un)installation.
\ No newline at end of file
+![Sematext Inventory Main Screen](../images/monitoring/inventory_main_view.png)
+
+### Inventory Info
+
+The Inventory Info information provides a summarized view of your inventory information allowing you to see:
+
+- Number of different packages in your monitored environment,
+- Number of different OS distributions,
+- Number of different OS versions,
+- A number of different kernel versions.
+
+![Inventory Info](../images/monitoring/inventory_inventory_info.png)
+
+### Inventory Overview
+
+The Inventory overview provides a per-host view of each host, virtual machine, and container along with the operating system and its version, number of CPUs, their type, memory, number of hard disk drives and total capacity.
+
+![Inventory Overview](../images/monitoring/inventory_overview.png)
+
+Expanding each row gives detailed information about the system including:
+
+- Unique identifier of the system,
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
```

Comment:
Maybe not "Hard" because they could also be SSDs. So just "Disk drive information".

Reply:
Good point, done.